### ISSUE TITLE
Suggestion: "mrsk run" to run on local dev machine

### ISSUE THREAD

**Issue by MadSpindel** [mrsked/mrsk#120]
I would like to have something like `mrsk deploy` but for local dev machine. It could be called `mrsk run`. It should download the image from container repository and then run it on local docker with same variables as it would be when running `mrsk deploy` against servers. It should spin up Traefik and everything on localhost.

The use case I'm looking for is to be able to try the image on local dev machine before deploying it to servers.

**Comment by acidtib** [comment]
You can already achieve this by spinning up virtual machines locally (for example, VirtualBox), then using the regular `mrsk deploy` to bring up your app locally.

**Comment by MadSpindel** [comment]
> You can already achieve this by spinning up virtual machines locally (for example, VirtualBox), then using the regular `mrsk deploy` to bring up your app locally.

Yes, but this is more about having a quick way of testing without the need to set up any virtual server or ssh key access. And in most use cases I believe this will only be used as a first attempt/evaluation on how to start using MRSK and to see how it works before deciding to go all-in on the tool.
